### PR TITLE
Require read rather than edit permissions to download bulk users

### PR DESCRIPTION
**Pull request by dannyroberts**
https://dimagi-dev.atlassian.net/browse/SAAS-10803

##### SUMMARY
Looks like this was just an oversight or that originally our permissions were less nuanced and this was never updated.

There's no way to use the bulk _download_ feature to edit users, right?

### CODE REVIEW

**Review by snopoke — APPROVED** [review]

**Review by esoergel — APPROVED** [review]
:boot: :construction:

**Comment by dannyroberts** [comment]
Realized that there are a small handful of different places in the UI where we do permissions checks and that merging this as is won't be useful until they're all done. Leaving the PR here as a good starting point, but I (or someone else) will probably pick up when we get to it

**Comment by biyeun** [comment]
@dannyroberts made some small changes. everything appears to be working locally, I'll rebuild and deploy staging since I see you have it in staging.yaml

File: corehq/apps/users/views/mobile/users.py [hunk 1]
```diff
@@ -1128,10 +1128,21 @@ def user_download_job_poll(request, domain, download_id, template="hqwebapp/part
     return render(request, template, context)
 
 
-class DownloadUsersStatusView(BaseManageCommCareUserView):
+class DownloadUsersStatusView(BaseUserSettingsView):
     urlname = 'download_users_status'
     page_title = ugettext_noop('Download Users Status')
 
+    @method_decorator(require_can_edit_or_view_commcare_users)
```
Comment (dannyroberts) [inline corehq/apps/users/views/mobile/users.py:1134]: I've seen `@method_decorator(require_can_edit_or_view_commcare_users, name='dispatch')` as a decorator on the class to do this in cases like this where there are no other changes to the context of the method itself. I think I like that better personally, but I don't know that we have a standard around that or anything

**Review by dannyroberts — COMMENTED** [review]
✅ (I'm not allowed to formally approve because github technically has me as the PR author)

**Pull request merged**